jberet-manifest/src/main/resources/jberet-manifest.yaml

```diff
@@ -0,0 +1,8 @@
+schemaVersion: "1.0.0"
+name: "jberet-manifest"
+id: "org.jberet:jberet-channel"
```

Comment:
This should be the GAV IMO so something like `org.jberet:jberet-manifest` or whatever you end up naming the artifact.